packages/orbit-components/src/OrbitProvider/index.tsx

```diff
@@ -14,9 +13,18 @@ import type { Props } from "./types";
 const getCssVarsForWL = (theme: typeof defaultTokens) =>
   Object.keys(theme).reduce((acc, key) => {
     if (key.startsWith("palette")) {
-      acc[key] = parseToRgba(theme[key] as string)
-        .slice(0, -1)
-        .join(", ");
+      try {
+        acc[key] = parseToRgba(theme[key] as string)
```

Comment:
wdyt, could we have some normalization here? kinda same as yours in Nitro, for removing whitespaces

Reply:
You mean, on top of the try/catch?

I guess it wouldn't hurt. 🤔 

@oreqizer what do you think?

Reply:
Yep, on top of it, just to avoid the same problem as reported and fixed elsewhere, I can imagine that it might occur, there might be unwanted whitespaces 👍🏻

Reply:
that is out of scope of Orbit though 😕 this is a Nest-specific bug with the whitespace. I wouldn't leak it into Orbit tbh

Reply:
ok, resolving then this 👍🏻